deploy.py

```diff
@@ -40,20 +40,21 @@ def open(name, *a, **kw):
         diff.update({b: False for b in remove})
     else:
         diff = {f: os.path.isfile(f) for f in diff}
-    deploy_data = {'subreddit': os.getenv('subreddit')}
-    for image, exists in diff.items():
-        data = dict(deploy_data)
-        with open(image) as image_file:
-            if exists:
-                data['image_path'] = image
-            else:
-                data['name'] = os.path.splitext(
-                    os.path.basename(image_file.name)
-                )[0]
-            getattr(
-                reddit,
-                "{0}_image".format("upload" if exists else "delete")
-            )(**data)
+    for subreddit in os.getenv('subreddit').split('+'):
+        deploy_data = {'subreddit': subreddit}
+        for image, exists in diff.items():
```

Comment:
Running for loops consecutively is slow cause CPython isn't a JIT compiler. Do you mind moving th ings around so the "subreddit" for loop occurs within the context manager, `data` initially being an empty dict instead of a copied version of `deploydata`, and then set 'subreddit' on it within that loop?